src/macaron/slsa_analyzer/checks/check_result.py

```diff
@@ -37,7 +110,36 @@ class Confidence(float, Enum):
     MEDIUM = 0.7
 
     #: A low confidence score.
-    LOW = 0.5
+    LOW = 0.4
```

Comment:
I wonder why we decided to update this value for LOW confidence score?

Reply:
I have changed the low value to improve the normalized scores.